crypto/nrf_oberon/CHANGELOG.rst

```diff
@@ -9,6 +9,120 @@ Changelog - nRF Oberon
 
 All notable changes to this project are documented in this file.
 
+nrf_oberon - 3.0.13
+*******************
+
+New version of the nrf_oberon library with the following changes.
+
+Added
+=====
+* Added Oberon PSA core, a heavily code-size optimized and efficient implementation of PSA core licensed for use on Nordic devices.
+* Added ocrypto APIs for PBKDF2 support (CMAC and HMAC using SHA-1, SHA-256).
+* Added ocrypto APIs for SPAKE2+ using ECC curve type secp256r1.
+* Added Oberon PSA crypto drivers as source distribution with extensive support:
+  - AEAD (AES CCM, AES GCM, ChaCha20/Poly1305)
+  - Cipher (Chacha20, AES CTR, AES CBC, AES CCM* and AES ECB)
+  - EC J-PAKE using ECC curve type secp256r1
+  - ECDH using ECC curve types secp224r1, secp256r1, secp384r1
+  - X25519
+  - ECDSA using ECC curve types secp224r1, secp256r1, secp384r1
+  - Ed25519
+  - HASH (SHA-1, SHA-224, SHA-256, SHA-384 and SHA-512)
+  - HKDF
+  - PBKDF2 using CMAC and AES-128
+  - PBKDF2 using HMAC and SHA-1 or SHA-256
+  - TLS 1.2 PRF functions
+  - HMAC, CMAC and CBC MAC
+  - RSA PKCS#1 1.5 and 2.1
+* Added custom Oberon PSA crypto drivers as source distribution:
+  - CTR_DRBG
+  - HMAC_DRBG
+  - SPAKE2+ using ECC curve type secp256r1
+  - SRP
+
+Removed
+=======
+* Removed binary distribution of Oberon PSA crypto drivers
+
+Library built against Mbed TLS version 3.3.0.
+
+Added the following Oberon crypto libraries for nRF9160, nRF53, nRF52, and nRF51 architectures.
```

Comment:
Strangely enough, we are using nRF9160 i.e a specifig SOC, but nRF53, i.e a series.
This is a maintenance problem for adding new SoCs. for example the new nrf9161.